### PR TITLE
Service jacket to CO vendor

### PR DESCRIPTION
**Pull request by NessiePendragon**
# About the pull request

Adds black service jacket to CO vendor.

# Explain why it's good for the game

XO has one in their vendor, so I added it to CO vendor as well for more drip options!

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Added service jacket to CO vendor.
/:cl:

### CODE REVIEW

**Comment by So-mii** [comment]
24 karat kino

File: code/modules/gear_presets/uscm_ship.dm [hunk 1]
```diff
@@ -534,7 +535,7 @@
 	name = "USCM Commanding Officer (CO++)"
 	idtype = /obj/item/card/id/general
 	paygrades = list(PAY_SHORT_MO6 = JOB_PLAYTIME_TIER_0)
-	dress_under = list(/obj/item/clothing/under/marine/dress/blues/senior, /obj/item/clothing/under/marine/dress/blues/general)
+	dress_under = list(/obj/item/clothing/under/marine/dress/blues/senior, /obj/item/clothing/under/marine/dress/blues/general, /obj/item/clothing/suit/storage/jacket/marine/dress)
```
Comment (nauticall) [inline code/modules/gear_presets/uscm_ship.dm:538]: jacket should be in `dress_over` instead of `dress_under` here

**Review by nauticall — CHANGES_REQUESTED** [review]

File: code/modules/gear_presets/uscm_ship.dm (hunk 1 above)
Comment (NessiePendragon) [reply code/modules/gear_presets/uscm_ship.dm:538]: oh, my bad. I will fix it

Comment (NessiePendragon) [reply code/modules/gear_presets/uscm_ship.dm:538]:
```suggestion
	dress_under = list(/obj/item/clothing/under/marine/dress/blues/senior, /obj/item/clothing/under/marine/dress/blues/general)
```

**Review by nauticall — APPROVED** [review]
there we go

**Pull request merged**